### ISSUE TITLE
Can't assign config file values with regex special characters

### ISSUE THREAD

**Issue by JBBee** [aphysci/gravity#214]
Setting `DiskUsageDeletionFilePattern="(.*(\d|DIR|IPN)\.bin)"` in a config file results in a parser error

**Comment by mbarger99** [comment]
@JBBee this was fixed in #215 right?  Not sure why this didn't auto-close.

**Comment by JBBee** [comment]
@mbarger99 yes, #215 closes two issues.  I think it's because I used the word `Fixes` instead of `Closes` in the PR

**Comment by mbarger99** [comment]
I think ["Fixes" is a valid keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), but maybe it didn't like the "for" in between that and the issue numbers??  The GitHub Issue settings says we've enabled auto-closing "linked" issues.  In any case, I'll close them.

Thanks for fixing them!
